tests/agents/test_agents.py

```diff
@@ -0,0 +1,183 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
@ashwinb @raghotham -- What's your thought on also moving standalone `agents/` scripts to pytests?

https://github.com/meta-llama/llama-stack-apps/tree/main/examples/agents